crates/krun-guest/src/pulse.rs

```diff
@@ -0,0 +1,21 @@
+use std::{
+    fs,
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+use anyhow::{Context, Result};
+
+pub fn setup_pulse_proxy(path: PathBuf) -> Result<()> {
+    if Path::new("/usr/bin/socat").exists() {
```

Comment:
WDYT about looking up `socat` from PATH? This would be useful for people who use `guix`, for example.

Reply:
Good idea, done!